### PR TITLE
Mention the relationship to Resource Bundles (issue #630)

### PR DESCRIPTION
**Pull request by hayatoito**
It'd be better to mention how this proposal and Resource Bundles are
related briefly.

We'll add more details and update alternative designs with some of ideas
from resource-bundles and others.

### CODE REVIEW

**Review by littledan — APPROVED** [review]
Thanks!

**Comment by hayatoito** [comment]
@jyasskin Could you please review this?

File: explainers/subresource-loading.md [hunk 1]
```diff
@@ -215,6 +215,17 @@ compressed](https://github.com/yoavweiss/url_compression_experiments).
 
 ## Alternate designs
 
+### Resource Bundles
```
Comment (jyasskin) [inline explainers/subresource-loading.md:218]: Remember to update the table of contents. I use https://marketplace.visualstudio.com/items?itemName=AlanWalk.markdown-toc to do that automatically.

**Review by jyasskin — APPROVED** [review]

File: explainers/subresource-loading.md (hunk 1 above)
Comment (hayatoito) [reply explainers/subresource-loading.md:219]: A good point. I forgot it. Let me update the TOC, then merge this PR. Thanks!

**Pull request merged**